### PR TITLE
Restore fullscreen button

### PR DESCRIPTION
**Pull request by little9**
This commit makes the fullscreen button visible
on the work page.

There is a feature spec that checks for this and
the absence of the download button.

chrome-driver was converting the `<br>`'s that
are also tested in the view_work spec into whitespace
so I moved those tests into a seperate file that doesn't
use chrome-driver.

There is a `manifest.json` in public that is used
in the testing environment so that we can get UV to load
correctly.

This also updates the .travis.yml to include chromium and
chromedriver and the Dockerfile as well.

Connected to UCLALibrary/ursus#278

### CODE REVIEW

File: app/views/catalog/_uv.html.erb [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <iframe
     class="universal-viewer-iframe"
+    <% if Rails.env == 'test' %>
```
[the remark this answers is not in the document]
Comment (val99erie) [reply app/views/catalog/_uv.html.erb]: Maybe it won't work to stub the entire template because that's what you are testing, but instead pull the `src` value into a helper method and then stub out that helper method for testing.

Let me know if that makes sense.

**Comment by jendiamond** [comment]
Screen Shot 2019-03-22 at 8 35 18 AM
